hw/bsp/dialog_da1469x-dk-pro/reset.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+ip=${1:-"localhost"}
+freq=${2:-"1000"}
+
+if [ -z $(which arm-none-eabi-gdb) ]; then
+  GDB=gdb
+else
+  GDB=arm-none-eabi-gdb
+fi
+
+GDB_CMD_FILE=${GDB_CMD_FILE:-".gdb_load"}
+OPENOCD_LOG_FILE=${OPENOCD_LOG_FILE:-".openocd_log"}
+CFG_FILE_DIR=${CFG_FILE_DIR:-"/srv/juul-test/jtest/openocd"}
+
+cat > ${GDB_CMD_FILE} << EOF
+set pagination off
+EOF
+if [ "$ip" == "localhost" ]; then
+cat >> ${GDB_CMD_FILE} << EOF
+shell sh -c "trap '' 2; openocd -f ${CFG_FILE_DIR}/ft4232h.cfg -f ${CFG_FILE_DIR}/steam.cfg -d > ${OPENOCD_LOG_FILE} 2>&1 &"
+EOF
+else
+cat >> ${GDB_CMD_FILE} << EOF
+shell sh -c "trap '' 2; ssh -tt osf@$ip docker-compose exec jtest \
+openocd -f ${CFG_FILE_DIR}/ft4232h.cfg -f ${CFG_FILE_DIR}/steam.cfg -f ${CFG_FILE_DIR}/bindto.cfg -d > .openocd_log 2>&1 &"
+EOF
+fi
+cat >> ${GDB_CMD_FILE} << EOF
+target remote ${ip}:3333
+mon adapter_khz ${freq}
+mon reset halt
+mon resume
+
+set *(int *)0x100C0050 = 1
```

Comment:
What does this do?

Reply:
HW RESET REQ. added a comment.

Reply:
> HW RESET REQ. added a comment.

Yeah, I checked the DS this morning, I think it would be possible to add a comment but it's OK anyway.